packages/cubejs-cli/src/config.ts

```diff
@@ -23,28 +33,30 @@ export class Config {
     await fs.writeJson(configFile, config);
   }
 
-  configFile() {
+  protected configFile() {
     const cubeCloudConfigPath = this.cubeCloudConfigPath();
     const configFile = path.join(cubeCloudConfigPath, 'config.json');
+
     return { cubeCloudConfigPath, configFile };
   }
 
-  async envFile(envFile) {
+  public async envFile(envFile: string) {
     if (await fs.pathExists(envFile)) {
       return dotenv.config({ path: envFile }).parsed;
     }
+
     return {};
   }
 
-  cubeEnvConfigPath() {
+  protected cubeEnvConfigPath() {
     return path.join(os.homedir(), '.env');
   }
 
-  cubeCloudConfigPath() {
+  protected cubeCloudConfigPath() {
     return path.join(os.homedir(), '.cubecloud');
   }
 
-  async deployAuth(url) {
+  public async deployAuth(url?: string) {
```

Comment:
all methods are public by default

Reply:
You are right, but when it's not written explicitly we can skip some places where methods, properties must be protected/private.

I enabled rule that will help us to protect this moment.

https://github.com/cube-js/cube.js/pull/1281/commits/f96c6e16312c8dc5471b447d3aeccf63d77d108f#diff-44ead173231c78d119ff39f7c132458a4cd7dde766706f8af36ae7b856ed149aR52

It's useful, example of detected problems.

https://github.com/cube-js/cube.js/pull/1281/commits/cf78849d061632057cbd660f8e100ff9b4752bd3#diff-e6ac146ee7d51883c4831298bc0c00a879996706e99b90b2cfc5139d2705440dR30